### PR TITLE
Added Feature Req. Issue Template & Updated to b10219 of 2023.1

### PR DESCRIPTION
**Pull request by PSGitHubUser1**
This PR will add Issue Template for Feature Request which I made and I am using it in my repos. But because you merged my prev. PR, this is a gift for you. As only few devs accepts my PRs :/ [Holi Special Gift] 

If you merged it then I will make PULL REQUEST Template, Yes, You heard right, I will make PR of PR Template.

_I know that you not even heard of 'Holi' word. Its an Huge Indian Festival which is on 8th March._

Btw check out #388

### CODE REVIEW

**Comment by PSGitHubUser1** [comment]
Updated to new version of 2023.1 [b10610]

File: src/plugin.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "id": "generator-fabric-1.19.2",
   "minversion": 202300108712,
-  "maxversion": 202300109612,
+  "maxversion": 202300110610,
```
Comment (Goldorion) [inline src/plugin.json:4]: why? this is a total regression

**Review by Goldorion — CHANGES_REQUESTED** [review]

**Review by Goldorion — CHANGES_REQUESTED** [review]
There is always one rule with GitHub. 1 PR = 1 feature, not 3.

File: update.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
   "generator-fabric-1.19.2": {
-    "latest": "1.8.6"
+    "latest": "1.8.7"
```
Comment (Goldorion) [inline update.json:3]: not sure if you check the master branch, but you don't.

**Comment by PSGitHubUser1** [comment]
> There is always one rule with GitHub. 1 PR = 1 feature, not 3.

Uhh. Ok, I did not knew that. Thanks.

**Pull request closed without merging**